identity-service/default-config.json

```diff
@@ -68,6 +68,7 @@
   "recaptchaServiceKey": "",
   "hCaptchaSecret": "",
   "solanaFeePayerWallet": "[]",
+  "solanaFeePayerWallets": "[]",
```

Comment:
seems like a dupe?

Reply:
not a dupe actually, there is logic for this e.g. here https://github.com/AudiusProject/audius-protocol/blob/master/identity-service/src/routes/healthCheck.js#L282